app/code/local/Magmodules/Webwinkelconnect/Model/Api.php

```diff
@@ -77,52 +77,48 @@ public function sendInvitation($order)
         $crontype = 'orderupdate';
-        $orderId = $order->getIncrementId();
         $storeId = $order->getStoreId();
         $apiId = trim(Mage::getStoreConfig('webwinkelconnect/general/api_id', $storeId));
         $apiKey = trim(Mage::getStoreConfig('webwinkelconnect/general/api_key', $storeId));
-        $delay = trim(Mage::getStoreConfig('webwinkelconnect/invitation/delay', $storeId));
-        $language = Mage::getStoreConfig('webwinkelconnect/invitation/language', $storeId);
-        $resendDouble = Mage::getStoreConfig('webwinkelconnect/invitation/resend_double', $storeId);
-
-        $email = $order->getCustomerEmail();
-        $customerName = $order->getCustomerName();
-
-        $url = 'https://www.webwinkelkeur.nl/api.php?id=' . $apiId;
-        $url .= '&password=' . $apiKey . '&email=' . urlencode($email) . '&order=' . $orderId;
-        $url .= '&delay=' . $delay . '&client=magento&customername=' . urlencode($customerName);
-
-        if (!$resendDouble) {
-            $url = $url . '&noremail=1';
-        }
-
-        if (!empty($language)) {
-            if ($language == 'cus') {
-                $lanArray = array('NL' => 'nld', 'EN' => 'eng', 'DE' => 'deu', 'FR' => 'fra', 'ES' => 'spa');
-                $address = $order->getShippingAddress();
-                if (isset($lanArray[$address->getCountry()])) {
-                    $url = $url . '&language=' . $lanArray[$address->getCountry()];
-                }
-            } else {
-                $url = $url . '&language=' . $language;
-            }
+        $postData['email'] = $order->getCustomerEmail();
+        $postData['order'] = $order->getIncrementId();
+        $postData['delay'] = intval(Mage::getStoreConfig('webwinkelconnect/invitation/delay', $storeId));
+        $postData['customer_name'] = $order->getCustomerName();
+        $postData['client'] = 'magento1';
+        $postData['platform_version'] = Mage::getVersion();
+        $postData['language'] = $this->getLanguage($storeId, $order);
+
+        if (Mage::getStoreConfig('webwinkelconnect/product_review_invites/enabled')) {
+            $postData['order_data'] = json_encode([
+                'order' => $order,
+                'products' => $this->getOrderProducts($order),
+            ]);
         }
 
-        $curl = curl_init($url);
-        curl_setopt($curl, CURLOPT_VERBOSE, 1);
-        curl_setopt($curl, CURLOPT_FAILONERROR, false);
-        curl_setopt($curl, CURLOPT_HEADER, 0);
-        curl_setopt($curl, CURLOPT_FOLLOWLOCATION, 1);
-        curl_setopt($curl, CURLOPT_RETURNTRANSFER, 1);
-        curl_setopt($curl, CURLOPT_SSL_VERIFYPEER, false);
-        $response = curl_exec($curl);
-        curl_close($curl);
+        $url = 'https://dashboard.webwinkelkeur.nl/api/1.0/invitations.json?' .
+            http_build_query([
+                'id' => $apiId,
+                'code' => $apiKey,
+            ]);
+
+        $curl = new Varien_Http_Adapter_Curl();
```

Comment:
Yeah, as I noted in another comment, I don't think the `Varien_Http_Adapter_Curl` is actually an improvement. My bad, I hadn't expected it to be like that.

What _would_ be an improvement in the original code would be to change the options above to this:

~~~php
        curl_setopt($curl, CURLOPT_FAILONERROR, true);
        curl_setopt($curl, CURLOPT_FOLLOWLOCATION, true);
        curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);
~~~